doc/getting_started/index.rst

```diff
@@ -273,27 +273,29 @@ to build Zephyr applications.
 
          .. code-block:: bash
 
-            cd ~
+            cd ~/zephyrproject
```

Comment:
Personally, I prefer to have my tools / programs outside any repository.

Especially as tools might be up-/downgrade independently of the repo.

Or I could have multiple up-/downstream workspaces, but just a single toolchain.

So I would suggest to still advice users with a different path.

Which of course also means the comment goes for remaining part of this commit.

Reply:
I am doing this on purpose for the getting started guide precisely to make the Zephyr environment more self-contained, so we appear to have different preferences in this regard. The question is really what will be easier for more people, I suppose.

Reply:
could you take a look at:
https://github.com/zephyrproject-rtos/sdk-ng/pull/199
https://github.com/zephyrproject-rtos/zephyr/pull/24143

those will remove the need for the environment variables, and in such way make it easier to get started.
And not polluting the Zephyr workspace with a toolchain folder.

Reply:
> could you take a look at:
> [zephyrproject-rtos/sdk-ng#199](https://github.com/zephyrproject-rtos/sdk-ng/pull/199)
> #24143
> 

OK, I like that PR very much. Let's wait until it gets reviewed by @nashif and @galak to decide how to proceed here.

Reply:
I suggest using either `/opt/` or ~/bin/` instead

Reply:
> I suggest using either `/opt/` or ~/bin/` instead


I think `~/bin` is a bad option, as the Zephyr SDK is not a single executable file, but several folders with executables, libraries, etc.

Currently, the suggested default paths to search for the SDK are:
```
$HOME/zephyr-sdk*
$HOME/.local/zephyr-sdk*
$HOME/.local/opt/zephyr-sdk*
/opt/zephyr-sdk*
/usr/zephyr-sdk*
/usr/local/zephyr-sdk*
```
so I suggest we pick one among those in the example given in gsg.

Feel free to comment on default paths for SDK in this PR:
https://github.com/zephyrproject-rtos/sdk-ng/pull/199

Reply:
I meant `~/bin/zephyr-sdk` @tejlmand